### ISSUE TITLE
Rodeo fails when another app is using localhost 3000

### ISSUE THREAD

**Issue by coristig** [yhat/rodeo#308]
Rodeo 1.4.3 fails to launch when I'm running another app on localhost:3000

**Comment by TakenPilot** [comment]
This is true!  It currently relies on port 3000, but I'm going to change that soon to randomly pick ports until it finds one.

It uses the server to serve up plots.  Images and html served up this way are treated differently by Chrome, memory-wise, compared to inlined images and inlined iframes.  This was done to prevent one of the things that used to crash the app.

I'm going to keep this bug open for our records until I've added the fix.

**Comment by TakenPilot** [comment]
Moved port to something more obscure.
